README.md

```diff
@@ -1,3 +1,10 @@
+#Team able-waterBuffalo -
+Lisa Brown, Sal Soloman, Abraham Ferguson
+
+#Goals:
```

Comment:
These goals are not enough for me to determine completion, should I be the person to review this.  A good starting point would be to fork the data-structures repository, and re-purpose the README for your specification and commitment (I think this already includes both quality and completeness guidelines).